egs2/TEMPLATE/s2t1/README.md

```diff
@@ -55,11 +55,23 @@ where `<sop>` is a special token denoting the start of prev/prompt sentence. The
 <sop> I'm going to talk today about energy and climate.<sos><en><transcribe><0.00> And that might seem a bit surprising, because my full-time work at the foundation is mostly about vaccines and seeds, about the things that we need to invent and deliver to help the poorest two billion live better lives.<14.12><15.36> But energy and climate are extremely important to these people; in fact, more important than to anyone else on the planet.<24.26><eos>
 ```
 
-During data preparation, three text files are generated:
+During data preparation, three text files should be generated:
 - `text` contains the normal target sentence, i.e., the text between `<sos>` and `<eos>`.
+    - This must include all special tokens as described above.
+    - If your data does not have timestamps, you need to use `<notimestamps>` instead. Even so, you still need to make sure that your non-linguistic symbol list (nlsyms_txt) contains [the range of all possible timestamps](https://github.com/espnet/espnet/blob/master/egs2/owsm_v1/s2t1/local/utils.py#L13-L19).
+    - `<sop>`, `<sos>`, and `<eos>` are actually inserted during preprocessing/dataloading. You only need `<language><task><starttime1> utt1<endtime1><starttime2> utt2<endtime2>`
+    - See https://github.com/espnet/espnet/tree/master/egs2/owsm_v1/s2t1/local for an example of data preparation and the expected format.
 - `text.prev` contains the previous sentence, i.e., the text between `<sop>` and `<sos>`. This might be unavailable at the beginning of a talk. In such cases, a special token `<na>` will be used.
+    - This should not contain any special tokens except for `<na>`. In the example above, take the text between `<sop>` and `<sos>` and put it here.
 - `text.ctc` contains the ASR transcript without any special token, which is used for the CTC loss. For ASR utterances, this can be derived from `text`, but for ST utterances, this is in a different language. If the ASR transcription is not available, `<na>` will be used.
-
+    - This should not contain any special tokens. Just take the text between `<task>` and `<eos>` and put it here (no timestamps).
```

Comment:
> Just take the text between `<task>` and `<eos>` and put it here (no timestamps).

For ASR, yes. But for ST, `text.ctc` is different text. `text.ctc` is the ASR transcript.

Reply:
good catch! just fixed